src/components/internal-header/internal-header.yaml

```diff
@@ -20,6 +20,10 @@ params:
   type: string
   required: false
   description: The public path for the assets folder. If not provided it defaults to /assets
+- name: useDeprecatedCrown
+  type: boolean
+  required: false
+  description: A feature flag to use old St Edward’s Crown. If "false" or unset, defaults to the Tudor Crown
```

Comment:
it's confusing because the idea then is that in play-frontend-hmrc we make components default to use the tudor crown unless there's some app config to turn it off - which we add to app-config-common before we release the play-frontend-hmrc version so that it can't go live until we remove that

Reply:
@oscarduignan On it.